docs/release-notes/saros-i_0.1.0.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Saros/I 0.1.0 Release Notes
+---
+
+This is the first alpha release of Saros/I, so expect it to be a bit rough around the edges.
+There are still some [restrictions](#restrictions) that apply to the usage and some basic [features are still missing](#missing-features).
+This release is not compatible with any other Saros plugin/version (like Saros/E).
+
+The GUI used in Saros/I is only a placeholder providing the minimal, necessary functionality.
```

Comment:
Just write that the current GUI will be replaced by a HTML GUI. The end user do not care about the internals.

Reply:
Done.